partner_event/models/res_partner.py

```diff
@@ -12,19 +12,18 @@ class ResPartner(models.Model):
     registrations = fields.One2many(
         string="Event registrations",
         comodel_name='event.registration', inverse_name="partner_id")
-    registration_count = fields.Integer(
-        string='Event registrations number', compute='_count_registration')
-    attended_registration_count = fields.Integer(
-        string='Event attended registrations number',
-        compute='_count_attended_registration')
-
-    @api.one
-    @api.depends('registrations')
-    def _count_registration(self):
-        self.registration_count = len(self.registrations)
+    event_count = fields.Integer(
+        string='Events',
+        compute='_compute_event_count',
+        oldname="registration_count",
+        help="Count of events with confirmed registrations.",
+    )
 
     @api.one
     @api.depends('registrations.state')
-    def _count_attended_registration(self):
-        self.attended_registration_count = len(self.registrations.filtered(
-            lambda x: x.state == 'done'))
+    def _compute_event_count(self):
+        self.event_count = len(
```

Comment:
Yes it will, but if there is a real performance problem there, it will affect not only this module, but all Odoo, and should be patched upstream. That's what I mean.